src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderBookingId.kt

```diff
@@ -0,0 +1,5 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.repository
+
+interface OffenderBookingId {
```

Comment:
this seemed the nicest way to just get the list of booking ids without loading the whole `OffenderBooking` for each record returned